x-pack/plugins/ml/public/jobs/jobs_list/components/job_details/forecasts_table/forecasts_table.js

```diff
@@ -151,65 +155,110 @@ class ForecastsTable extends Component {
     if (forecasts.length === 0) {
       return (
         <EuiCallOut
-          title="No forecasts have been run for this job"
+          title={(<FormattedMessage
+            id="xpack.ml.jobsList.jobDetails.forecastsTable.noForecastsTitle"
+            defaultMessage="No forecasts have been run for this job"
+          />)}
           iconType="iInCircle"
         >
           <p>
-            To run a forecast,
-            open the <EuiLink onClick={() => this.openSingleMetricView()}>Single Metric Viewer</EuiLink>
+            <FormattedMessage
+              id="xpack.ml.jobsList.jobDetails.forecastsTable.noForecastsDescription"
+              defaultMessage="To run a forecast, open the {singleMetricViewerLink}"
```

Comment:
This doesn't look to be displaying the link as it should:

![image](https://user-images.githubusercontent.com/7405507/48558524-608dee00-e8e1-11e8-92c3-760ff6ebc6e4.png)

Reply:
please check one more time, this issue was related to pseudo locale. we have fixed it and it should work good now.

Reply:
The first time I opened the Forecasts tab up, I still saw the wrong text, but after that the link was rendered correctly.